src/test/java/se/sundsvall/invoicesender/integration/db/DbIntegrationTests.java

```diff
@@ -54,6 +56,36 @@ void testGetBatches() {
 		verifyNoMoreInteractions(batchRepositoryMock);
 	}
 
+	@Test
+	void testGetBatchesByStatusReady() {
+		var ready = READY;
+
+		var batchEntity = createBatchEntity();
+		var batchEntity2 = createBatchEntity();
+		var expectedBatch = List.of(batchEntity, batchEntity2);
+
+		when(batchRepositoryMock.findAllByBatchStatus(ready)).thenReturn(expectedBatch);
+
+		var actualBatch = dbIntegration.getBatchesByStatus(ready);
+
+		assertThat(expectedBatch).isEqualTo(actualBatch).hasSize(2);
+	}
+
+	@Test
+	void testGetBatchesByStatusManaged() {
+		var managed = MANAGED;
```

Comment:
No need to define this as a local variable - just use the enum constant directly.